docs/test-specifications/schedule-service/ScheduleSignTransaction.md

```diff
@@ -0,0 +1,119 @@
+| 3       | Sign a schedule without providing any signatures                    | scheduleId=<VALID_SCHEDULE_ID>                                                          | Fails with `NO_NEW_VALID_SIGNATURES`. **Verification**: ScheduleInfo signatories array unchanged.                                                                   | N                 |
+| 4       | Sign a schedule that requires multiple signatures (first signature) | scheduleId=<MULTI_SIG_SCHEDULE_ID>, commonTransactionParams.signers=[<SIGNER_KEY_1>]    | Transaction succeeds, signature added to schedule. **Verification**: ScheduleInfo signatories array contains new entry; executed_timestamp remains null.            | N                 |
+| 5       | Sign a schedule that requires multiple signatures (all signatures)  | scheduleId=<MULTI_SIG_SCHEDULE_ID>, commonTransactionParams.signers=[<SIGNER_KEY_2>]    | Transaction succeeds, schedule executed immediately. **Verification**: ScheduleInfo executed_timestamp is set; signatories array contains both expected entries.    | N                 |
+| 6       | Sign a schedule with duplicate signature                            | scheduleId=<ALREADY_SIGNED_SCHEDULE_ID>, commonTransactionParams.signers=[<SAME_KEY>]   | Fails with `NO_NEW_VALID_SIGNATURES`. **Verification**: ScheduleInfo signatories array unchanged.                                                                   | N                 |
+| 7       | Sign a schedule with key list (partial fulfillment)                 | scheduleId=<KEYLIST_SCHEDULE_ID>, commonTransactionParams.signers=[<KEYLIST_KEY_1>]     | Transaction succeeds, signature added to schedule. **Verification**: ScheduleInfo signatories array contains new entry; executed_timestamp remains null.            | N                 |
+| 8       | Sign a schedule with key list (complete fulfillment)                | scheduleId=<KEYLIST_SCHEDULE_ID>, commonTransactionParams.signers=[<KEYLIST_KEY_2>]     | Transaction succeeds, schedule executed immediately. **Verification**: ScheduleInfo executed_timestamp is set; signatories array contains all required key entries. | N                 |
+| 9       | Sign a schedule with insufficient transaction fee                   | scheduleId=<VALID_SCHEDULE_ID>, commonTransactionParams=[maxFee=<LOW_FEE>]              | Fails with `INSUFFICIENT_TX_FEE`. **Verification**: ScheduleInfo signatories array unchanged.                                                                       | N                 |
+| 10      | Sign a schedule with invalid account ID                             | scheduleId=<VALID_SCHEDULE_ID>, commonTransactionParams=[payerAccountId=<INVALID_ID>]   | Fails with `INVALID_ACCOUNT_ID`. **Verification**: ScheduleInfo signatories array unchanged.                                                                        | N                 |
```

Comment:
```suggestion
| 10      | Sign a schedule with invalid account ID                             | scheduleId=<VALID_SCHEDULE_ID>, commonTransactionParams.transactionId=<INVALID_ID>   | Fails with `INVALID_ACCOUNT_ID`. **Verification**: ScheduleInfo signatories array unchanged.                                                                        | N                 |
```
`commonTransactionParams.transactionId` specs say you can just feed in account ID to specify payer